client/landing/stepper/declarative-flow/update-options.ts

```diff
@@ -12,14 +15,19 @@ const updateOptions: Flow = {
 	useSteps() {
 		return [ STEPS.OPTIONS, STEPS.PROCESSING, STEPS.ERROR ];
 	},
+	useSideEffect() {
+		const { setIntent } = useDispatch( ONBOARD_STORE );
 
+		useEffect( () => {
+			setIntent( Onboard.SiteIntent.UpdateOptions );
+		}, [] );
+	},
 	useStepNavigation( currentStep, navigate ) {
-		const flowName = this.name;
 		const siteSlug = useSiteSlug();
 		const flowToReturnTo = useQuery().get( 'flowToReturnTo' ) || 'free';
 
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do we need this?

Reply:
There are alternatives. But what do you suggest?

These sort of "fixes" are usually a sign of the code that underlies it. In this case, you really didn't need the two arguments signature with a spread on the second. A single object would have been better

Reply:
My bad, I didn't know `providedDependencies` was not used at all underneath. I thought it was a mistake.

Reply:
Testing now..